### ISSUE TITLE
Moving to an organization

### ISSUE THREAD

**Issue by patrick96** [polybar/polybar#1464]
So a few months ago I created the organization @polybar with the idea that at some point we could migrate all polybar repositories into that organization.

This has a few benefits:
* **All core devs have full access privileges to all repos**
We don't need to ping jaagr every time we need some settings adjusted
* **Better privilege management via teams and outside collaborators**
With [teams](https://help.github.com/articles/about-teams/) we can give new collaborators access to just those repos they are responsible for. Some use cases for this could be for adding new maintainers for just i3ipcpp, xpp or polybar.github.io. The same thing for [outside collaborators](https://help.github.com/articles/adding-outside-collaborators-to-repositories-in-your-organization/).
* Having all polybar repos bundled in one place

Some of these benefits may not seem like a lot now, but in the future I can see the polybar project having even more repos (for external modules once we finish the plugin architecture, documentation, tooling) and more maintainers with different levels of access.

The migration would be done with the [Transfer Ownership](https://help.github.com/articles/transferring-a-repository-owned-by-your-personal-account/) feature. I have done some testing and believe such a migration could be done without breaking anything. I will do a writeup of what I tested and how exactly the migration process will happen after we all agreed to proceed with the migration.

With this migration I would also like make polybar more formally *our* (skystrife, jaagr, NBonaparte, patrick96) project instead of just jaagr's. This is already de facto the case but I want to make it more official. I want polybar to stay alive, even if some people become inactive and this will help with that. jaagr will still retain the copyright to the project unless we decide to create a legal entity for this project, but I think this is a bit overkill right now. 

@jaagr, @skystrife, @NBonaparte, do you support migrating the repos [jaagr/polybar](https://github.com/jaagr/polybar), [jaagr/i3ipcpp](https://github.com/jaagr/i3ipcpp), [jaagr/xpp](https://github.com/jaagr/xpp) to the [polybar organization](https://github.com/polybar)?

@jaagr, I want to hear especially from you, because in the end this hinges on you. With this migration polybar would officially stop being just *your* project but would become *our* project (including you). Are you OK with that?

**Comment by NBonaparte** [comment]
It would be nice to centralize everything, but as you said, it's up to jaagr to approve it.

**Comment by skystrife** [comment]
I'm fine with this, but also agree that it's up to @jaagr.

**Comment by jaagr** [comment]
Of course 👍

**Comment by patrick96** [comment]
### Test Results:
So, as I said before, I tested out the "Transfer Ownership" feature with a [sample repo](https://github.com/polybar/migration-test).

Here are the results:

**What is migrated?**
Basically everything. git history, issues, PRs, milestones, projects, wiki, releases, settings.

**Are links to the old repo updated?**
No. The transfer doesn't change any content. All links still point to the old URL. Issues and PRs referenced with `#` also point to the old URL, those links are only updated if the line they're on is changed (github seems to cache the generated html from comments).

**How do old links work?**
All links to the old repo URL will be redirected to the same relative URL in the new repo. shields.io badges still work properly. And more importantly if the moved repo is a submodule somewhere, that submodule still works.

### Migration:
I'd like to do the migration in multiple steps, just in case something goes wrong:

* [ ] Transfer ownership of [jaagr/xpp](https://github.com/jaagr/xpp)
* [ ] Update xpp submodule to new URL
* [ ] Transfer ownership of [jaagr/i3ipcpp](https://github.com/jaagr/i3ipcpp)
* [ ] Update i3ipcpp submodule to new URL
* [ ] Transfer ownership of [jaagr/polybar](https://github.com/jaagr/polybar)
* [ ] Update all links to jaagr/* repos inside the repo (shields.io badges, PKGBUILD, config, manpage, zsh completions). Essentially there should be no links left that contain `jaagr` in any of the repos.
* [ ] Update all outside links to the old URL (wiki, AUR, reddit, irc, polybar.github.io)

The reason I want to do it this way is that we can catch any errors before we have fully migrated.
@jaagr, you only need to do the "Transfer Ownership" part, everything else can be done by me/us.

# Important
@jaagr under no circumstance can you *ever* create a repo with the name `polybar`, `xpp`, or `i3ipcpp` in your account. If you do, all links to the old URLs will be broken. This also means you can never press the 'Fork' button on any of these three repos. This is a bit of an inconvenience since you cannot have a functional fork in your own account and can thus not create PRs (there is no way when you don't have a fork). So, if you are working on a new branch, create that branch in the polybar/polybar repo instead.

Please read [Transferring a repository owned by your personal account](https://help.github.com/articles/transferring-a-repository-owned-by-your-personal-account/#transferring-to-an-organization) for how to transfer the repo to the org. Also post it here, once you have transferred a repo so I can do the next step and give you the all clear for the next transfer.

For me to update the subreddit and irc I need permissions in those places, as mentioned in #1489 my username in reddit is `patrick96MC`, on IRC in order for me to set the topic and the url I need at least the flags `+s` and `+t`, but it's probably best to give me the same access that you have `+AFORefiorstv`. This can be done with `/msg ChanServ flags #polybar patrick96 +AFORefiorstv`.

I really hope I haven't missed anything.

**Comment by TiagoDanin** [comment]
If you want I can move the [Awesome Polybar](https://github.com/TiagoDanin/Awesome-Polybar) to the organization.

**Comment by patrick96** [comment]
Wow, I didn't even know about Awesome-Polybar!

We will discuss this once we have migrated our own repos.

@x70b1 from [polybar-scripts](https://github.com/x70b1/polybar-scripts) also offered to migrate.

**Comment by patrick96** [comment]
Migration is done and there haven't been any issues so far.

We have also decided to migrate @x70b1's polybar-scripts repo.

@TiagoDanin We found that if we migrated both your repo and x70b1's, we would have a lot of redundant content since you both link to some of the same pages and you even link to some of his. We decided to go with x70b1 because he is more active in the community, especially on IRC. But I have linked to your project on our wiki.
